src/NServiceBus.NHibernate/Outbox/OutboxCleaner.cs

```diff
@@ -36,6 +37,25 @@ protected override Task OnStart(IMessageSession busSession)
                 throw new Exception("Invalid value in \"NServiceBus/Outbox/NHibernate/FrequencyToRunDeduplicationDataCleanup\" AppSetting. Please ensure it is a TimeSpan.");
             }
 
+            var key = "NServiceBus/Outbox/NHibernate/TimeToWaitBeforeTriggeringCriticalErrorWhenCleanupTaskFails";
```

Comment:
I actually inlined this variable to be consistent with how other configuration keys are handled. Key values seem to be descriptive and only used locally in this file. Since they don't change often (doco), encapsulating them brings little value.

Agree?